test/functionalTest/cases/2248_forwarding_updates_overwrite_the_types_of_attributes/forwarding_updates_overwrite_the_types_of_attributes.test

```diff
@@ -139,12 +141,18 @@ Date: REGEX(.*)
 
 
 03. Update E1/T1/A1 in CB, using PUT /v2/entities/E1/attrs/A1/value, without stating type of the attribute, value is a string
+Note: we get 404 Not Found, but is ok. It is due to accumulator-server.py doesn't answers in NGSIv1
 =============================================================================================================================
-HTTP/1.1 204 No Content
-Content-Length: 0
+HTTP/1.1 404 Not Found
+Content-Length: 61
+Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
+{
```

Comment:
Case 2 should be covered by update_three_found_on_cps_two_local_four_not_found_in_ngsiv2.test, see steps 13 and 14:

    # 13. Register E2/T1/A4 in CB, provApp: CP3 (E2-A4 does not exist in CP3)
    # 14. Update/APPEND E1/T1/A4 in CB

Wait, this is for an attribute that doesn't exist. You are asking about an entire Entity ...
No, I can't recall we have such a test.
Very easy to add a new test for it though